SPECS/openssh.spec

```diff
@@ -653,6 +651,9 @@ getent passwd sshd >/dev/null || \
 %endif
 
 %changelog
+* Tue Apr 30 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.2 + 0.10.3-2
+- Remove build dependency on xauth (used for X11 forwarding not supported on XCP-ng hosts)
+
 * Mon Apr 29 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.1 + 0.10.3-2
 - Cleanup spec file
 - Harden default ciphers and algorithms
```

Comment:
As the previous build failed and this new PR is necessary to make it build, there should be only one changelog entry with all the changes. No need to bump the release either.

Reply:
Fixed